pkg/apiserver/experiment/experiment.go

```diff
@@ -373,16 +373,13 @@ func (s *Service) createStressChaos(exp *core.ExperimentInfo, kubeCli client.Cli
 					Mode:     v1alpha1.PodMode(exp.Scope.Mode),
 					Value:    exp.Scope.Value,
 				},
+				ContainerNames: exp.Target.StressChaos.ContainerNames,
```

Comment:
Panic here if `exp.Target.StressChaos.CintainerName` is nil.

Reply:
Will be an empty array because of the `omitempty` option?

Reply:
Got it

Reply:
you are right